vignettes/ard.Rmd

```diff
@@ -0,0 +1,150 @@
+---
+title: "ARD"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{ARD}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+```{r setup, message=FALSE, warning=FALSE, echo=FALSE}
+library(tfrmt)
+library(cards)
+library(dplyr)
+library(tidyr)
+library(pharmaverseadam)
+library(gt)
+```
+
+# How to prepare input ARD
```

Comment:
Let's add a bit first about how we expect an ARD as input, and what an ARD is. Feel free to reference materials here: https://www.danieldsjoberg.com/ARD-RinPharma-workshop-2024/slides/01-intro-cards/#/title-slide

Reply:
added some more detail let me know what you think now :)